python/ml4ir/applications/classification/model/losses/categorical_cross_entropy.py

```diff
@@ -18,18 +18,15 @@ def get_loss(loss_key) -> RelevanceLossBase:
         raise NotImplementedError
 
 
-class CategoricalCrossEntropy(RelevanceLossBase):
-
-    def get_loss_fn(self, **kwargs):
+class CategoricalCrossEntropy(losses.CategoricalCrossentropy, RelevanceLossBase):
```

Comment:
what is the advantage of defining it here? Consistency with the rest of loss definitions? Wouldn't it work if we imported from keras directly?

Reply:
The way we have defined loss functions in ml4ir, they are coupled with the final activation layer. You can make the case that we shouldn't do this as default and that's a fair argument. But it will need some minor redesign.